dangerfile.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @name Production PR Dangerfile
+ *
+ * @description
+ * This dangerfile validates production PRs before the deployment step
+ * to assure all change order requirements are met.
+ * The data captured here is then forwarded to Azure.
+ *
+ * See the orb commands for reference:
+ */
+const danger = require('danger')
+const { checkAssignees } = require('danger-plugin-complete-pr')
```

Comment:
This is quite funny. I was searching around for this file and was confused. Then realized this is a "thing" https://www.npmjs.com/package/danger-plugin-complete-pr